### PR TITLE
feat:  new wallet page

### PR DESCRIPTION
**Pull request by nuanyang233**

### CODE REVIEW

File: packages/dashboard/src/components/ConnectActionList/icons.tsx
Comment (septs) [inline packages/dashboard/src/components/ConnectActionList/icons.tsx:3]: use https://aka.pw/svgomg reduce the image.

File: packages/dashboard/src/pages/Wallets/icons.tsx
Comment (septs) [inline packages/dashboard/src/pages/Wallets/icons.tsx:3]: use https://aka.pw/svgomg reduce the image.

**Comment by Jack-Works** [comment]
@nuanyang233 Netlify CI failed, please check

File: packages/dashboard/src/components/DashboardFrame/index.tsx
Comment (nuanyang233) [inline packages/dashboard/src/components/DashboardFrame/index.tsx:32]: Just meet the design size

File: packages/dashboard/src/components/CreateWalletForm/index.tsx
Comment (Jack-Works) [inline packages/dashboard/src/components/CreateWalletForm/index.tsx:64]: You can use makeStyles for now.

Comment (nuanyang233) [reply packages/dashboard/src/components/CreateWalletForm/index.tsx:64]: Ok, actually there is nothing to do with makeStyled and styled here, the problem I encountered is that I need to set the property on MuiFilledInput-root, but MUI does not provide the rule name to set.

Comment (Jack-Works) [reply packages/dashboard/src/components/CreateWalletForm/index.tsx:64]: Isn't there anything like `<FormControl InputProps={{ classes: { root: {} } }} />`?

Comment (nuanyang233) [reply packages/dashboard/src/components/CreateWalletForm/index.tsx:64]: No, not provided here

File: packages/dashboard/src/components/DashboardFrame/index.tsx
Comment (Jack-Works) [reply packages/dashboard/src/components/DashboardFrame/index.tsx:32]: please leave a proper comment

File: packages/dashboard/src/components/Mnemonic/MnemonicConfirm.tsx
Comment (Jack-Works) [inline packages/dashboard/src/components/Mnemonic/MnemonicConfirm.tsx:17]: Is this component mobile friendly?

File: packages/dashboard/src/components/Mnemonic/MnemonicReveal.tsx
Comment (Jack-Works) [inline packages/dashboard/src/components/Mnemonic/MnemonicReveal.tsx:31]: is this comp mobile ready?

File: packages/dashboard/src/components/WalletConnect/index.tsx
Comment (Jack-Works) [inline packages/dashboard/src/components/WalletConnect/index.tsx:13]: Avoid hardcode color

File: packages/dashboard/src/components/WalletConnect/index.tsx
Comment (Jack-Works) [inline packages/dashboard/src/components/WalletConnect/index.tsx:14]: Avoid. Use Typography

File: packages/dashboard/src/components/WalletConnect/index.tsx
Comment (Jack-Works) [inline packages/dashboard/src/components/WalletConnect/index.tsx:22]: Is it mobile friendly?

File: packages/dashboard/src/pages/Wallets/icons.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/icons.tsx:1]: Please move those icons to /src/icons/

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:16]: Mobile?

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:20]: Mobile? theme.spacing

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:25]: Mobile?

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:23]: Mobile?

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:29]: Mobile?

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:31]: Avoid hardcode color

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:30]: Avoid odd number. theme.spacing

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:37]: Mobile; Theme.spacing

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:42]: Mobile; Theme.spacing

File: packages/dashboard/src/pages/Wallets/ImportWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/ImportWallet.tsx:53]: Mobile; Theme.spacing

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:18]: Mobile

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:30]: Mobile; avoid hardcode color and spacing; avoid hardcode font-size;

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:36]: avoid hardcode color and spacing; avoid odd padding number

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:45]: mobile friendly

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:51]: mobile friendly; avoid odd spacing; avoid hardcode color

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:56]: mobile friendly; avoid odd spacing;

File: packages/dashboard/stories/ConnectWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/stories/ConnectWallet.tsx:6]: Hi I suggest to move page stories to `stories/pages/...`

File: packages/dashboard/stories/MnemonicConfirm.tsx
Comment (Jack-Works) [inline packages/dashboard/stories/MnemonicConfirm.tsx:7]: suggest to use @storybook/actions instead of argTypes. Provide it in args bellow. it's more typed

**Review by Jack-Works — CHANGES_REQUESTED** [review]
@nuanyang233 Hi I think we should focus on the curent review suggestions instead of add more code. After all suggestions resolved I'll merge this and we can continue to work on this on a new PR. This make review easier.

If you found my change request is not easy to done or not applicable, please PM me

File: packages/dashboard/src/components/WalletQRCodeContainer/index.tsx
Comment (Jack-Works) [inline packages/dashboard/src/components/WalletQRCodeContainer/index.tsx:8]: Emm I think for this effect, this CSS is quite scary. Maybe an SVG can be better.

Comment (Jack-Works) [reply packages/dashboard/src/components/WalletQRCodeContainer/index.tsx:8]: hmm maybe this is fine. never mind

**Comment by Jack-Works** [comment]
it seems like you're converting all components to desktop only. Don't do that if they _can_ support mobile easily.

**Comment by nuanyang233** [comment]
> it seems like you're converting all components to desktop only. Don't do that if they _can_ support mobile easily.

Ok, I revert them and only kept Mnemonic, I think this component is not convenient for users to operate on mobile, I will modify it if the designer thinks it is necessary to keep them on mobile. Other components mobile support I will modify when I have mobile design draft

**Comment by Jack-Works** [comment]
hi @nuanyang233 plz fix the linter error and rebase this pr. I have just merged #2475

**Comment by nuanyang233** [comment]
> hi @nuanyang233 plz fix the linter error and rebase this pr. I have just merged #2475

cc

**Comment by Jack-Works** [comment]
@nuanyang233 it still contains changes in the #2475

File: packages/dashboard/src/pages/Wallets/icons.tsx
Comment (Jack-Works) [reply packages/dashboard/src/pages/Wallets/icons.tsx:1]: cc @nuanyang233

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:90]: Also wrong. The correct way is to split value(what stored in the state) and label (what displayed to the user)

File: packages/dashboard/src/pages/Wallets/CreateWallet.tsx
Comment (Jack-Works) [inline packages/dashboard/src/pages/Wallets/CreateWallet.tsx:94]: No need to memo those labels

**Review by Jack-Works — APPROVED** [review]
let's wait for the CI passes

**Pull request merged**
